src/CMakeLists.txt

```diff
@@ -16,3 +16,12 @@ target_link_libraries(polylog
     polylog_cpp
     polylog_fortran
 )
+
+
+FILE(GLOB HEADERS ${CMAKE_CURRENT_SOURCE_DIR}/*.h ${CMAKE_CURRENT_SOURCE_DIR}/*.hpp)
+set_target_properties(polylog PROPERTIES PUBLIC_HEADER "${HEADERS}")
+include(GNUInstallDirs)
+install(TARGETS polylog polylog_c polylog_cpp polylog_fortran
+  PUBLIC_HEADER 
+  DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/polylog
```

Comment:
I think we should name the sub-directory as the package name, i.e. `polylogarithm`, not `polylog`. (I'm afraid that `polylog` may be confusing or may conflict with another package.)

Reply:
But the `.a`/`.so` should keep their (current) name of just polylog?

Reply:
Good point. To be consistent (and not confusing) I think library should be called `libpolylogarithm.{a,so,dylib,...}`. This may require more changes to the CMake files, but I'd say that this is ok in order to be consistent.